accepted/future-releases/class-modifiers/feature-specification.md

```diff
@@ -935,6 +984,15 @@ needed.
 
 ## Changelog
 
+1.6
+
+* Update the modifiers applied to anonymous mixin applications to closer
+  match the superclass/mixin modifiers.
+* State that `enum` declarations count as `final`.
+* Some rephrasing to allow concept reuse.
+* Say that pre-feature libraries can mix in  non-`mixin` platform library classes
```

Comment:
Maybe add link to https://dart-review.googlesource.com/c/sdk/+/287665 here?

Reply:
Did so in the section describing the exceptions.